website/templates/emails/pending_registration_admin.html.mako

```diff
@@ -11,16 +11,40 @@
     ${initiated_by} has initiated a registration of your project ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
     % endif
     <br>
-    If approved, a registration will be created for the project and will be made public immediately.<br>
+    If approved, a registration will be created for the project and will be made public immediately
+    % if is_moderated:
+         and sent to ${reviewable.provider.name} moderators for review
+    % endif
+    .
```

Comment:
Is there going to end up being a space between the last word of the sentence and this period?